cloudinstall/utils.py

```diff
@@ -264,12 +264,14 @@ def render_charm_config(config):
         install_type=config.getopt('install_type'),
         openstack_password=config.getopt('openstack_password'))
 
-    if config.getopt('openstack_release'):
```

Comment:
@battlemidget looks like I led you astray here. I guess we need this if here, because in the test suite, there aren't always defaults. Here's the test failure:
```
======================================================================
ERROR: test_render_openstack_release_default (test_utils.TestRenderCharmConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/python/3.4.3/lib/python3.4/unittest/mock.py", line 1136, in patched
    return func(*args, **keywargs)
  File "/home/travis/build/Ubuntu-Solutions-Engineering/openstack-installer/test/test_utils.py", line 80, in test_render_openstack_release_default
    self._do_test_osrel(None, None, mockspew)
  File "/home/travis/build/Ubuntu-Solutions-Engineering/openstack-installer/test/test_utils.py", line 67, in _do_test_osrel
    render_charm_config(self.config)
  File "/home/travis/build/Ubuntu-Solutions-Engineering/openstack-installer/cloudinstall/utils.py", line 273, in render_charm_config
    config.getopt('openstack_release'))
TypeError: Can't convert 'NoneType' object to str implicitly
```
Although -- maybe the right thing to do here is just fix the tests.